specs/trust-store-trust-policy.md

```diff
@@ -206,7 +210,7 @@ The following table shows the resultant validation action, either *enforced* (ve
 
 **Revocation check** : Guarantees that the signing identity is still trusted at signature verification time. Events such as key or system compromise can make a signing identity that was previously trusted, to be subsequently untrusted. This guarantee typically requires a verification-time call to an external system, which may not be consistently reliable. The `permissive` verification level only logs failures of revocation check and does not enforce it. If a particular revocation mechanism is reliable, use `strict` verification level instead.
 
-- **NOTE** : `notation` RC1 will not have in built support for Revocation Check feature, see [Key Management - Rescinding Signatures (CRL)](https://github.com/notaryproject/notaryproject/issues/72) for details. When this feature is subsequently implemented, the `strict` verification level will **enforce** this validation, and will fail signatures which would previously pass signature verification when revocation checks were not supported. This is considered a **breaking change**, but is the appropriate behavior for a `strict` verification level.
+- **NOTE** : `notation` RC2 will not have in built support for Revocation Check feature, see [Key Management - Rescinding Signatures (CRL)](https://github.com/notaryproject/notaryproject/issues/72) for details. When this feature is subsequently implemented, the `strict` verification level will **enforce** this validation, and will fail signatures which would previously pass signature verification when revocation checks were not supported. This is considered a **breaking change**, but is the appropriate behavior for a `strict` verification level.
```

Comment:
Good point, I've removed this section from here and will create new PR in notation to call out its limitation.

Reply:
I agree with you to remove it, since it is more about implementation progress, not specification itself. I did notice that there is a release for this repo https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-rc.1, which is also called rc.1. Will this cause confusion that the spec rc.1 matched the notation rc.1?

I also agree with you that we should put the limitations in both Readme.md and release notes. In the future, we should also link the release note or create a different format in notaryproject website. Currently we don't have release blogs yet.  see example https://oras.land/blog/oras-0.14-and-future/

Reply:
> I did notice that there is a release for this repo https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-rc.1, which is also called rc.1. Will this cause confusion that the spec rc.1 matched the notation rc.1?

IMO, that should not be an issue. Spec version and implementation versions can (and will) diverge, its implementations responsibility to call out which spec version it supports.

Reply:
Tracking it in  https://github.com/notaryproject/notaryproject/issues/233